packages/@okta/vuepress-theme-prose/assets/css/base/_typography.scss

```diff
@@ -9,13 +9,102 @@
   font-style: normal;
 }
 
+@font-face {
+  font-family: 'ABC Whyte';
```

Comment:
ABC Whyte is not rendering for me in the Netlify build. Looks like these URLs are 404ing.